services/route_manager/src/main/java/com/futurewei/alcor/route/exception/VpcContainsSubnet.java

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.route.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code= HttpStatus.CONFLICT, reason="there are some subnets exist in the VPC. We cannot delete VPC router and VPC default routing table.")
+public class VpcContainsSubnet extends Exception{
+}
```

Comment:
VpcContainsSubnet => VpcNonEmptyException

"there are some subnets exist in the VPC. We cannot delete VPC router and VPC default routing table."
=> VPC router and route tables can't be deleted as the associated VPC contains subnets

Reply:
Agree